lib/modules/shared/widgets/loading_indicator.widget.dart

```diff
@@ -12,11 +12,14 @@ class LoadingIndicator extends StatelessWidget {
       case LoadingIndicatorType.Regular:
         return CircularProgressIndicator();
       case LoadingIndicatorType.Button:
+      case LoadingIndicatorType.OutlinedButton:
         return SizedBox(
           width: ThemeConfig.of(context).buttonHeight / 2,
           height: ThemeConfig.of(context).buttonHeight / 2,
           child: CircularProgressIndicator(
-            color: ThemeConfig.of(context).onPrimaryColor,
+            color: type == LoadingIndicatorType.Button
+                ? ThemeConfig.of(context).onPrimaryColor
+                : ThemeConfig.of(context).primaryColor,
```

Comment:
aici de ce mai verifici daca type == LoadingIndicatorType.Button? 
Esti deja in case-u de OutlinedButton, nu cred ca poate sa aiba alta valoare :))

Reply:
Aici sunt în 2 case-uri, de Button și OutlinedButton; am zis să nu le tratez total separat, pentru că returnează cam același lucru, se schimbă doar culoarea :))